.travis.yml

```diff
@@ -30,7 +30,7 @@ virtualenv:
 install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
-  - pip install ofxparse
+  - pip install git+https://github.com/jseutter/ofxparse.git
```

Comment:
Can you do this in requirements.txt if still needed?

Reply:
@pedrobaeza But this already was in .travis.yml. I just updated the link to a working version. But if you think it better to create a new requirements.txt in the repository, I will do it.

Reply:
OK, no special problem. It's just the new convention for being clearer.